src/docs/docs.yaml

```diff
@@ -197,6 +287,96 @@ definitions:
           is_verified:
             type: boolean
             default: false
+          role:
+            type: string
+            default: requester
+          manager_id:
+            type: integer
+            nullable: true
+          gender:
+            type: string
+            nullable: true
+          date_of_birth:
+            type: string
+            format: date
+            nullable: true
+          department:
+            type: string
+            nullable: true
+          preferred_language:
+            type: string
+            nullable: true
+          residential_address:
+            type: string
+            nullable: true
+          preferred_currency:
+            type: string
+            nullable: true
+          createdAt:
+            type: string
+            format: date-time
+          updatedAt:
+            type: string
+            format: date-time
+  UsersResponse:
+    properties:
+      status:
+        type: string
+      data:
+        type: object
+        properties:
+          id:
+            type: integer
+          email:
+            type: string
+            format: email
+          first_name:
+            type: string
+          last_name:
+            type: string
+          is_admin:
+            type: boolean
+            default: false
+          is_verified:
+            type: boolean
+            default: false
+          role:
+            type: string
+            default: requester
+          manager_id:
+            type: integer
+            nullable: true
+          gender:
+            type: string
+            nullable: true
+          date_of_birth:
+            type: string
+            format: date
+            nullable: true
+          department:
+            type: string
+            nullable: true
+          preferred_language:
+            type: string
+            nullable: true
+          residential_address:
+            type: string
+            nullable: true
+          preferred_currency:
```

Comment:
do we have `residential_address` and `preferred_currency` in our user model?

Reply:
Yes, we do

Reply:
I added it to the model and migrations in my branch

Reply:
Okay